make-release.sh

```diff
@@ -268,5 +264,4 @@ fi
 
 if [[ $CREATE_PULL_REQUESTS == "true" ]]; then
   createPRToXBranch
-  createPRToMasterBranch
```

Comment:
Don't remove it.
We will port released files into master branch.

Reply:
ok